remote-backup/run.sh

```diff
@@ -162,9 +162,9 @@ function copy-backup-to-remote {
     fi
 
     bashio::log.info "Copying backup using SFTP/SCP."
-    if ! sshpass -p "${REMOTE_PASSWORD}" scp -s -F "${SSH_HOME}/config" "/backup/${SLUG}.tar" remote:"${remote_directory}/${remote_name}.tar"; then
+    if ! sshpass -p "${REMOTE_PASSWORD}" scp -s -F "${SSH_HOME}/config" "/backup/${SLUG}.tar" "${REMOTE_HOST}":"${remote_directory}/${remote_name}.tar"; then
```

Comment:
That change is incorrect, `remote` refers to the entry in the generated config file!